### PR TITLE
Allow more succinct `Requires`

### PR DESCRIPTION
**Pull request by Crisfole**
It's bugged me for a long time that I can't list my claims out like so:

    this.RequiresClaims("Claim One", "Claim Two");
    this.RequiresAnyClaim("Claim One", "Claim Two");

If this gets merged in I *will* be able to do that.

### CODE REVIEW

File: src/Nancy/Security/ModuleSecurity.cs [hunk 1]
```diff
@@ -32,6 +32,16 @@ public static void RequiresClaims(this INancyModule module, IEnumerable<string>
         }
 
         /// <summary>
+        /// This module requires authentication and certain claims to be present.
+        /// </summary>
+        /// <param name="module">Module to enable</param>
+        /// <param name="requiredClaims">Claim(s) required</param>
+        public static void RequiresClaims(this INancyModule module, params string[] requiredClaims)
+        {
+            module.RequiresClaims((IEnumerable<string>)requiredClaims);
```
Comment (khellang) [inline src/Nancy/Security/ModuleSecurity.cs]: What about `requiredClaims.AsEnumerable()`?

File: src/Nancy/Security/ModuleSecurity.cs [hunk 2]
```diff
@@ -41,6 +51,16 @@ public static void RequiresAnyClaim(this INancyModule module, IEnumerable<string
...
         }
+        
+        /// <summary>
+        /// This module requires authentication and any one of certain claims to be present.
+        /// </summary>
+        /// <param name="module">Module to enable</param>
+        /// <param name="requiredClaims">Claim(s) required</param>
+        public static void RequiresAnyClaim(this INancyModule module, params string[] requiredClaims)
+        {
+            module.RequiresAnyClaim((IEnumerable<string>)requiredClaims);
```
Comment (khellang) [inline src/Nancy/Security/ModuleSecurity.cs]: Same here :smile:

**Comment by Crisfole** [comment]
You know, one of those methods that never showed up on my radar. I'll go make the switch.

**Comment by Crisfole** [comment]
Done.

**Comment by Crisfole** [comment]
And now I *even* know where to find it.

**Pull request merged**